include/brod.hrl

```diff
@@ -17,104 +17,55 @@
 -ifndef(__BROD_HRL).
 -define(__BROD_HRL, true).
 
--type kafka_key()                 :: kpro:key().
```

Comment:
Perfect. Then one final question remains, do. you want the references from the specs reverted considering they're taken from brod_int.hrl or are you fine with them referring to brod:some_type()? 

(As in you want to keep the types in brod.hrl for external clients I presume?)